build-scripts/verify_references.py

```diff
@@ -132,6 +129,15 @@ def get_profileruleids(xccdftree, profile_name):
     return ruleids
 
 
+def is_remote_feed(href):
+    return href.startswith("http://") or \
+            href.startswith("https://") or \
+            href.startswith("security-data-oval-com.redhat.rhsa-") or \
+            href.startswith("security-data-oval-com.oracle") or \
```

Comment:
In OL 7 and OL 8 data streams it's  `href="security-oval-com.oracle.elsa-all.xml.bz2"`
```suggestion
            href.startswith("security-oval-com.oracle") or \
```